### ISSUE TITLE
Ctrl+Shift+2 does nothing on Windows 10 Desktop or browser connection to Server

### ISSUE THREAD

**Issue by ronblum** [rstudio/rstudio#6461]
### System details

    RStudio Edition : Desktop & Server
    RStudio Version : 1.3.910
    OS Version      : Windows 10
    R Version       : 3.6.x

### Steps to reproduce the problem

1) On Windows 10, use RStudio Desktop or connect to RStudio Server with Firefox or Chrome
2) Ctrl+Shift+2

### Describe the problem in detail

Nothing happens.

### Describe the behavior you expected

The Console pane should be maximized.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [ ] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
No repro for me. RStudio desktop 1.3.923 on Windows-10 works, as does Server in Chrome.

**Comment by gtritchie** [comment]
Dup to an issue you logged a while ago: #4233

**Comment by francisbarton** [comment]
Just to note that I have the same problem as the OP (RStudio Desktop 1.2.5001, Windows 10).
I'm not aware of anything else in Windows binding to Ctrl-Shift-2.
In RStudio when I press this key combination, nothing happens.
